bedrock/firefox/views.py

```diff
@@ -511,13 +511,23 @@ class WhatsnewView(L10nTemplateView):
         "firefox/whatsnew/whatsnew-fx99-fr.html": ["firefox/whatsnew/whatsnew"],
         "firefox/whatsnew/whatsnew-fx99-en.html": ["firefox/whatsnew/whatsnew"],
         "firefox/whatsnew/whatsnew-fx99-en-rally.html": ["firefox/whatsnew/whatsnew"],
+        "firefox/whatsnew/whatsnew-fx101-vpn-en.html": ["firefox/whatsnew/whatsnew"],
+        "firefox/whatsnew/whatsnew-fx101-vpn-de.html": ["firefox/whatsnew/whatsnew"],
+        "firefox/whatsnew/whatsnew-fx101-vpn-fr.html": ["firefox/whatsnew/whatsnew"],
+        "firefox/whatsnew/whatsnew-fx101-vpn-mobile-free-en.html": ["firefox/whatsnew/whatsnew", "firefox/whatsnew/whatsnew-101-vpn-mobile"],
+        "firefox/whatsnew/whatsnew-fx101-vpn-mobile.html": ["firefox/whatsnew/whatsnew", "firefox/whatsnew/whatsnew-101-vpn-mobile"],
```

Comment:
Suggestion (blocking): the first Fluent file in the list will be used to determine activation, so should these be switched?
```suggestion
        "firefox/whatsnew/whatsnew-fx101-vpn-mobile-free-en.html": ["firefox/whatsnew/whatsnew-101-vpn-mobile", "firefox/whatsnew/whatsnew"],
        "firefox/whatsnew/whatsnew-fx101-vpn-mobile.html": ["firefox/whatsnew/whatsnew-101-vpn-mobile", "firefox/whatsnew/whatsnew"],
```